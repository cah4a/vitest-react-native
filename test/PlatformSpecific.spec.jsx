import { Platform } from './src/Platform';
import { test, expect } from 'vitest'
import { render } from '@testing-library/react-native'

test('platform specific renders correctly', () => {
  const app = render(<Platform />)
  expect(app).toMatchSnapshot()
})
