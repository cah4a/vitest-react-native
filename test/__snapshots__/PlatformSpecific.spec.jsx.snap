// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`platform specific renders correctly 1`] = `
<Text>
  iOS
</Text>
`;
